shared/profile/add-to-team/container.js

```diff
@@ -109,6 +112,7 @@ export default compose(
   ),
   lifecycle({
     componentDidMount() {
+      this.props.clearAddUserToTeamsResults()
       this.props.loadTeamList()
```

Comment:
could this just be a side effect of load

Reply:
Can't do that: we need to reload the team list immediately after getting an error, so that the text next to each checkbox stays correct for the case where some adds succeeded and some failed, so that would dismiss the error immediately.

Reply:
could be a flag to load

Reply:
There's another reason not to do this: when you toggle any checkbox while seeing an error, we want to clear errors (since they no longer apply to your selection) but not reload teams.  So I think it's correct that they're separate actions.